src/Resolver/AbstractInjection.php

```diff
@@ -1,5 +1,5 @@
 <?php
-
+// phpcs:ignoreFile
```

Comment:
Why should this file be ignored from CS?

Reply:
Phpcs fails because of the deprecation warning.

Reply:
Maybe we should just disable the rule then instead of disabling the whole file from being checked.